internal/v2/s2av2_e2e_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	accessTokenEnvVariable = "S2A_ACCESS_TOKEN"
-	testAccessToken        = "test_access_token"
+	testAccessToken        = "valid_token"
```

Comment:
Is this variable still needed?

Reply:
No, it's not, since I'm setting a different token in each Test, thanks for catching this!